pennylane/gradients/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This subpackage contains quantum gradient transforms."""
+import pennylane as qml
+
+from . import finite_difference
+
+from .finite_difference import finite_diff, finite_diff_stencil, generate_shifted_tapes

```

Comment:
Just a thought in terms of naming: would a user then use the following?
```python
qml.gradients.finite_difference.finite_diff
```
Would it make sense to name `finite_diff` such that it refers to what the function itself does? E.g., `get_tapes_and_processing_fn`? The information about the method (finite difference) seems to be duplicated.

(On the other hand, doing
```python
from pennylane.gradients.finite_difference import finite_diff
```
and calling `finite_diff` will make the user code more readable :thinking: )

Reply:
Ah, this function is imported top-level, so there is no module context!

It is currently accessible via
```python
>>> qml.gradients.finite_diff(tape)
```